ekglib/ldap_parser/parser.py

```diff
@@ -591,6 +601,21 @@ def _parse_entry_status(self):
         status = self.entry.entry_status
         self._add((self.entry_iri, LDAP.entryStatus, Literal(status)))
 
+    _substitution_iri_map = {
```

Comment:
this and the next seem stateless, suggest making them free-standing.

Reply:
pls bear with me, I am literally a python virgin :-P